src/main/java/com/minecolonies/coremod/entity/ai/citizen/builder/ConstructionTapeHelper.java

```diff
@@ -49,17 +50,18 @@ public static int checkIfPlaceable(@NotNull int x, @NotNull int y, @NotNull int
         int newY = y;
         boolean working = true;
         while (working)
-
         {
             final BlockPos block = new BlockPos(x, newY, z);
             final BlockPos blockMin1 = new BlockPos(x, newY-1, z);
-            if (world.getBlockState(block).isFullBlock() || world.getBlockState(block).getMaterial().isLiquid())
-                {newY = newY+1;}
-            else
-                if (world.getBlockState(blockMin1).isFullBlock() || world.getBlockState(blockMin1).getMaterial().isLiquid())
-                    {working = false;}
+            if (world.getBlockState(block).getMaterial().isReplaceable())
+                {
+                    if (world.getBlockState(blockMin1).getMaterial().isReplaceable())
+                        {newY = newY-1;}
```

Comment:
I'm sorry but this formats really weird :O
Can we change that to normal formatting?